{{cookiecutter.project_name}}/.github/workflows/run-tests.yml

```diff
@@ -77,9 +77,12 @@ jobs:
               }
             ]
           run-name: {{ cookiecutter.project_name}} Integration Test
+          {%- if cookiecutter.include_feature_store != "yes" %}
```

Comment:
Nit: But what if we make this `== "no"` instead? Bit cleaner code and easier for our customers to understand